app/src/main/java/com/mensinator/app/CalendarScreen.kt

```diff
@@ -91,9 +91,9 @@ fun CalendarScreen() {
     var previousFirstPeriodDate by remember { mutableStateOf<LocalDate?>(null) }
     val colorMap = ColorSource.getColorMap(isDarkMode())
 
-    val initPeriodMessage = dbHelper.getSettingByKey("period_notification_message")?.value.toString()
-    val periodMessageKey = ResourceMapper.getStringResourceId(initPeriodMessage)
-    val periodMessageText = periodMessageKey?.let { stringResource(id = it) } ?: initPeriodMessage
+    // Trigger notification with custom message
+    val initPeriodMessage = dbHelper.getStringSettingByKey("period_notification_message")
```

Comment:
Please change this from `initPeriodMessage` to `initPeriodKeyOrCustomMessage`

Reply:
changed in `SettingScreen.kt` as well to match